ral-facilities/datagateway
# Only display session expiry warning message once #1087

## Description
- Stores a value in local storage when the session expiry message is displayed and does not show it again until something is added to the selections after the cart has been completely emptied
- Also changes the text in the warning message to "Please remember that your selection will be lost after 2 hours of inactivity on this site"
![image](https://user-images.githubusercontent.com/90245114/151353310-b960b090-3952-4724-be32-1f7987dbd9cf.png)
- Prevents a `console.error` message being printed in citation formatter unit tests.

## Testing instructions
Ensure the message does not reappear when navigating between plugins and that removing all selections in download resets the message.


## Agile board tracking
Closes #1087, Closes ral-facilities/scigateway#916
